ext/QuantumOpticsExt/QuantumOpticsExt.jl

```diff
@@ -1,13 +1,14 @@
 module QuantumOpticsExt
 
 using QuantumInterface, QuantumOpticsBase
+using QuantumInterface: samebases
 using QuantumSymbolics
 using QuantumSymbolics:
     HGate, XGate, YGate, ZGate, CPHASEGate, CNOTGate, PauliP, PauliM,
     XCXGate, XCYGate, XCZGate, YCXGate, YCYGate, YCZGate, ZCXGate, ZCYGate, ZCZGate,
     XBasisState, YBasisState, ZBasisState,
     NumberOp, CreateOp, DestroyOp,
-    FockBasisState,
```

Comment:
This is a breaking change, one that might even already affect QuantumSavory. Could you check if this is used in QuantumSavory? If it is not, then everything is fine (although it should this be accompanied by a breaking version bump and a note about breakage in the changelog)